common/autoware_universe_utils/include/autoware/universe_utils/ros/managed_transform_buffer.hpp

```diff
@@ -58,12 +58,69 @@ struct PairEqual
   }
 };
 using TFMap = std::unordered_map<Key, Eigen::Matrix4f, std::hash<Key>, PairEqual>;
+constexpr std::array<const char *, 3> warn_frames = {"map", "odom", "world"};
```

Comment:
nit: this is a personal trauma, but would it be possible to use `std::string_view` for the constexpr?